public/src/components/channelManagement/headerTests/headerTestEditor.tsx

```diff
@@ -0,0 +1,211 @@
+import { useStyles } from '../helpers/testEditorStyles';
+
+interface HeaderTestEditorProps {
+  test: HeaderTest;
+  hasChanged: boolean;
+  onChange: (updatedTest: HeaderTest) => void;
+  onValidationChange: (isValid: boolean) => void;
+  visible: boolean;
+  editMode: boolean;
+  onDelete: () => void;
+  onArchive: () => void;
+  onTestSelected: (testName: string) => void;
+  testNames: string[];
+  testNicknames: string[];
+  createTest: (newTest: HeaderTest) => void;
+}
+
+const HeaderTestEditor: React.FC<HeaderTestEditorProps> = ({
+  test,
+  onChange,
+  onValidationChange,
+  visible,
+  editMode,
+  onArchive,
+  onDelete,
+  onTestSelected,
+  testNames,
+  testNicknames,
+  createTest,
+}: HeaderTestEditorProps) => {
+  const classes = useStyles();
+
+  const setValidationStatusForField = useValidation(onValidationChange);
+
+  const updateTest = (updatedTest: HeaderTest): void => {
+    onChange({ ...updatedTest });
+  };
+
+  const onVariantsSplitSettingsValidationChanged = (isValid: boolean): void =>
+    setValidationStatusForField('variantsSplitSettings', isValid);
+
+  const onControlProportionSettingsChange = (
+    controlProportionSettings?: ControlProportionSettings,
+  ): void => updateTest({ ...test, controlProportionSettings });
+
+  const onLiveSwitchChange = (isOn: boolean): void => {
+    updateTest({ ...test, isOn });
+  };
+
+  const onVariantsChange = (updatedVariantList: HeaderVariant[]): void => {
+    updateTest({ ...test, variants: updatedVariantList });
+  };
+
+  const onVariantChange = (updatedVariant: HeaderVariant): void => {
+    onVariantsChange(
+      test.variants.map(variant =>
+        variant.name === updatedVariant.name ? updatedVariant : variant,
+      ),
+    );
+  };
+
+  const onVariantDelete = (deletedVariantName: string): void => {
+    onVariantsChange(test.variants.filter(variant => variant.name !== deletedVariantName));
+  };
+
+  const onRegionsChange = (updatedRegions: Region[]): void => {
+    updateTest({ ...test, locations: updatedRegions });
+  };
+
+  const onCohortChange = (updatedCohort: UserCohort): void => {
+    updateTest({ ...test, userCohort: updatedCohort });
+  };
+
+  const onCopy = (name: string, nickname: string): void => {
+    onTestSelected(name);
+    createTest({ ...test, name: name, nickname: nickname, isOn: false });
+  };
+
+  const renderVariantEditor = (variant: HeaderVariant): React.ReactElement => (
+    <HeaderTestVariantEditor
+      key={`head-${test.name}-${variant.name}`}
+      variant={variant}
+      onVariantChange={onVariantChange}
+      onDelete={(): void => onVariantDelete(variant.name)}
+      editMode={editMode}
+      onValidationChange={(isValid: boolean): void =>
+        setValidationStatusForField(variant.name, isValid)
+      }
+    />
+  );
+
+  const renderVariantSummary = (variant: HeaderVariant): React.ReactElement => (
+    <TestEditorVariantSummary
+      name={variant.name}
+      testName={test.name}
+      testType="BANNER"
```

Comment:
`HEADER`

Reply:
Fixed, alongside adding 'HEADER' to the TestType type in helpers/shared.ts file